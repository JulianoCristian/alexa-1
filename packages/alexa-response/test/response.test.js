import Response from '../src';

test('chaining', () => {
  const response = Response.say('Hello')
                           .card({ title: 'Title', content: 'Content' })
                           .shouldEndSession(false)
                           .reprompt('Reprompt')
                           .build();

  expect(response).toMatchSnapshot();
});
