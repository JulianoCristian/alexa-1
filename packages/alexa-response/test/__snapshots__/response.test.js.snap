// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chaining 1`] = `
Object {
  "response": Object {
    "card": Object {
      "content": "Content",
      "title": "Title",
      "type": "Simple",
    },
    "outputSpeech": Object {
      "text": "Hello",
      "type": "PlainText",
    },
    "reprompt": Object {
      "outputSpeech": Object {
        "text": "Reprompt",
        "type": "PlainText",
      },
    },
    "shouldEndSession": false,
  },
  "version": "1.0",
}
`;
